### DISCUSSION TITLE
so-test seemingly runs but no pcap data in SO Console

### DISCUSSION THREAD

**Discussion by HundleBun** [Security-Onion-Solutions/securityonion#7097] [Unsupported Versions] [answered]
After running "so-test" it appears to run fine, with no errors in /opt/so/log/tcpreplay/init.log that I can see, however when I open the SO Console there is no pcap data to parse through...

SO Version: 2.3.100, Evaluation on-prem deployment (ESXi), Installed from Security Onion ISO

so-status shows all services are running "OK"

No failures on SOC Grid and "sudo salt-call state.highstate" completes with no failures

![image](https://user-images.githubusercontent.com/75502993/152400117-02462435-2725-4431-9444-5bae51a08b8f.png)

![image](https://user-images.githubusercontent.com/75502993/152400178-167f3320-913b-44f7-8b81-7a7db5f42590.png)

**Comment by HundleBun** [comment]
![image](https://user-images.githubusercontent.com/75502993/152400549-4de714af-47bf-4c0e-8368-383424ee0293.png)
Results of running so-test on the VM

**Comment by dougburks** [comment, marked as answer]
I just tested and it works correctly for me:

![Screen Shot 2022-02-03 at 1 49 28 PM](https://user-images.githubusercontent.com/1659467/152409300-34b8204b-9a37-47e7-9a5b-33c8bde67f1d.png)

Check your Hunt interface and see if there are any logs at all. 

If not, check `/nsm/zeek/logs/` to see if there are any logs there. 

If not, check to see if your sniffing interface is working properly. 

If all else fails and you just want to quickly test Security Onion, try an Import installation as shown at the link below and then import pcap files:
https://docs.securityonion.net/en/2.3/first-time-users.html

**Reply by HundleBun** [reply]
I do have logs in my Hunt interface for activity on the Management VM and SSH sessions I've used from the Analyst VM and some other logs. I also have logs populating from using "so-import-pcap" since "so-test" wasn't working. 

However I'm still not seeing any logs correlating to the pcap data that should be coming from "so-test". Since "so-import-pcap" is working it's not a terribly big deal but not knowing what is taking place to cause issues irks me.. The purpose of this VM was just to poke around and follow along with training videos to familiarize myself with SO and it's tools.

**Reply by dougburks** [reply]
If you want to investigate further, you could try some of the steps from my previous response:

If not, check `/nsm/zeek/logs/` to see if there are any logs there.

If not, check to see if your sniffing interface is working properly.
